play_tester.py

```diff
@@ -1,4 +1,5 @@
 # import source code
+from collections import Counter
```

Comment:
This looks like it may have been auto-imported by VS Code, since I'm not seeing it used in the file. I recommend turning that feature off in the VS Code settings so it doesn't auto-import libraries you're not expecting. 

In the future, if you make changes to a sandbox file like this, you don't need to add it to the staging area/commit it and push up the changes.

Totally ok that you did so here since this isn't a PR that will get merged into the project, but if your changes were to be merged into a project then someone reviewing your code would probably ask you to remove these changes from the PR so they don't get merged into the master code base.